src/otx/core/model/module/action_detection.py

```diff
@@ -31,31 +34,25 @@ def __init__(
         torch_compile: bool,
         optimizer: list[OptimizerCallable] | OptimizerCallable = lambda p: torch.optim.SGD(p, lr=0.01),
         scheduler: list[LRSchedulerCallable] | LRSchedulerCallable = torch.optim.lr_scheduler.ConstantLR,
+        metric: MetricCallable | None = None,
```

Comment:
Why does it cannot have the default MetricCallable and should be `None` as default?

Reply:
For example, at the export, predict, we don't need add metric.

Reply:
It's not necessary for things like exports, but wouldn't it be more convenient for API users to at least have this as the default matric when creating a model?
I think the first case of user scenario is to try training first.